src/features/onboarding/Onboarding/LastDateInput.js

```diff
@@ -31,7 +31,7 @@ const LastDateInput = ({ onChange, value, inputProps = {} }) => {
         format="MMMM do"
         id="date-picker-inline"
         value={value}
-        label="First day of your last period"
+        label={label || "First day of your last or current period"}
```

Comment:
**nitpick:** Fix width

![CleanShot 2021-05-12 at 08 57 23@2x](https://user-images.githubusercontent.com/478531/117932117-4beb0f00-b300-11eb-9807-9a218cd09b3e.png)

Reply:
I can obviously not spell this early in the day 🤣

Reply:
Thing is, this has no with, so to fix it I would have to put a width :(

Is that okay with you?

Reply:
Weird, but yeah I guess.

Reply:
I figured out why, it was because of how it did the transforms and stuff, it got smaller and moved, some css magic :/

Fixed with width of 104%